CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Add metrics in the following database insturmentation libraries. (#1973)
```

Comment:
Should we call out (e.g. by names) the metrics which are added?

@theletterf @MrAlias